src/org/python/core/PyFunction.java

```diff
@@ -518,6 +518,8 @@ public Object __tojava__(Class<?> c) {
                 String name = null;
                 for (Method method : c.getMethods()) {
                     if (method.getDeclaringClass() != Object.class) {
+                        if (method.isDefault())
```

Comment:
We have braces around all bodies (in any code less than about 15 years old). For ease of reading, I would do this as:
```
if (...) {
    continue;
} else if (...) {
    ...
} else {
    ...
}
```